manifests/config.pp

```diff
@@ -35,103 +36,180 @@
   $jsp_jar = 'jsp-2.1-6.0.2.jar'
   $jsp_url = "http://uk.maven.org/maven2/jetty/jsp/2.1-6.0.2/${jsp_jar}"
 
-  # This works for versions < 5.0 
+  # SOLR 3.x and 4.x
   if versioncmp($::solr::version, '5.0') < 0 {
 
-    #Copy the jetty config file
-    file { "/etc/default/${jetty_package}":
-      ensure  => file,
-      content => template('solr/jetty-default.erb'),
-      require => Package[$jetty_package],
-    }
+    case $::lsbdistcodename {
```

Comment:
I wouldn't have done it this way...
us the os version number and do a versioncmp so that if 
os version >= 18.04 do it the 'bionic' way, etc... provides some future proofing until the next LTS release that needs it installing in a different way...

Reply:
think the variable/fact is $::os.release.major

Reply:
@MWilkinson - Updated. Also simplified params.pp logic.